docs/releasing-new-versions.md

```diff
@@ -0,0 +1,168 @@
+# Releasing New Versions
```

Comment:
FYI @nerrad has some changes to this doc in #1655

Reply:
👍 Will need to resolve merge conflicts depending on whats first :)